lib/ansible/modules/cloud/amazon/redshift.py

```diff
@@ -276,36 +281,40 @@ def create_cluster(module, redshift):
               'number_of_nodes', 'publicly_accessible',
               'encrypted', 'elastic_ip', 'enhanced_vpc_routing'):
         if p in module.params:
-            params[p] = module.params.get(p)
+            # https://github.com/boto/boto3/issues/400
+            if module.params.get(p) is not None:
```

Comment:
Rather than doing `if p in module.params:` on [line 283](https://github.com/ansible/ansible/pull/37052/files#diff-fd4728adea42973936ca9bdb0082deadR283) you could move this line `if module.params.get(p) is not None:` there and unindent the line below.